src/components/analyticVisualization/VisualizationSearch/periods.js

```diff
@@ -1,9 +1,9 @@
-export const TODAY = 'today' || 'thisDay'
+export const TODAY = 'today' && 'thisDay'
```

Comment:
this will evaluate to 'thisDay', which I guess will work in the sense that `thisDay` is the relative period property on the visualizations list of relative periods, but if the goal is to get it to accept both `today` and `thisDay`, I think you would need to change the logic here to be something like

```
export const TODAY = 'today'
export const THIS_DAY = 'thisDay'
```

and then include both in your array of relative periods.